components/accelerator/nvidia/hw-slowdown/component.go

```diff
@@ -124,11 +125,15 @@ func (c *component) States(ctx context.Context) ([]components.State, error) {
 		}, nil
 	}
 
+	// Extract data source information
+	dataSourcesInfo := getDataSourceInfo(events)
+
 	return []components.State{
 		{
 			Name:    StateKeyHWSlowdown,
 			Healthy: false,
-			Reason:  fmt.Sprintf("hw slowdown events frequency per minute %.2f (total events per minute count %d) exceeded threshold %.2f for the last %s", freqPerMin, len(eventsByMinute), c.stateHWSlowdownEventsThresholdFrequencyPerMinute, c.stateHWSlowdownEvaluationWindow),
+			Reason: fmt.Sprintf("hw slowdown events frequency per minute %.2f (total events per minute count %d) exceeded threshold %.2f for the last %s (data sources: %s)",
```

Comment:
should we remove the `data sources`?  Make this sentence more natural

Reply:
renamed to `event source`